### ISSUE TITLE
question: Error: Cannot get connection "default" from the connection manager.

### ISSUE THREAD

**Issue by matheusriios** [typeorm/typeorm-typedi-extensions#43]
```ts
Error: Cannot get connection "default" from the connection manager. Make sure you have created such connection. Also make sure you have called useContainer(Container) in your application before you established a connection and importing any entity.
    at Object.value (/src/decorators/InjectConnection.ts:12:23)
    at /app/src/ContainerInstance.ts:355:37
    at Array.map (<anonymous>)
    at ContainerInstance.initializeParams (/app/src/ContainerInstance.ts:352:27)
    at ContainerInstance.getServiceValue (/app/src/ContainerInstance.ts:305:47)
    at ContainerInstance.get (/app/src/ContainerInstance.ts:119:21)
    at Object.value (/app/src/decorators/Inject.ts:48:42)
    at /app/src/ContainerInstance.ts:355:37
    at Array.map (<anonymous>)
    at ContainerInstance.initializeParams (/app/src/ContainerInstance.ts:352:27)
[nodemon] app crashed - waiting for file changes before starting...
```

My index.ts

```
import 'reflect-metadata';

import { Container } from 'typedi';
import { useContainer } from 'typeorm';
import Application from 'src/Application';

useContainer(Container);

const application = Container.get(Application);

application.start();
```

**Comment by ShinteiMai** [comment]
Same issue, the error only show up when I'm executing a GraphQL query/mutation from a resolver but there was no error on build/compilation time.

**Comment by NoNameProvided** [comment]
The error says it, you attempt to access the connection before you create in TypeORM via calling `createConnection`. Closing this issue as solved. If your issue still persists please open a new issue with a **minimal, reproducible** example.
